src/coreclr/vm/jitinterface.cpp

```diff
@@ -11822,6 +11873,103 @@ void* CEEJitInfo::getFieldAddress(CORINFO_FIELD_HANDLE fieldHnd,
     return result;
 }
 
+bool CEEInfo::getReadonlyStaticFieldValue(CORINFO_FIELD_HANDLE fieldHnd, uint8_t* buffer, int bufferSize)
+{
+    CONTRACTL {
+        THROWS;
+        GC_TRIGGERS;
+        MODE_PREEMPTIVE;
+    } CONTRACTL_END;
+
+    _ASSERT(fieldHnd != NULL);
+    _ASSERT(buffer != NULL);
+    _ASSERT(bufferSize == sizeof(uint64_t));
```

Comment:
This is a bit odd to me, can the EE not just have a check below to return false if the buffer isn't of the exact right size (will require small JIT change)?

Reply:
Thanks, done